README.hbs

```diff
@@ -8,13 +8,18 @@ node-dota2
 
 A node-steam plugin for Dota 2, consider it in alpha state.
 
+## Contributing
 Check out @RJacksonm1's blog post (his only blog post), [Extending node-dota2](https://blog.rjackson.me/extending-node-dota2/), for a rough overview of adding new functionality to the library.
 A fair warning, while the way you search for new functionality is still the same, quite a lot has changed (and been simplified) implementation wise.
 It is now easier to implement new functionality than it was back when this blog was written.
 
+Be careful, you'll need SVN if you want to work on the project! Executing an `npm install` inside the project directory
```

Comment:
I wonder if we can just pull the protobufs from steamdatabase via curl/wget like we do for the dota2 specific protobufs

Reply:
Well that's what we did before, but with steam moving to steam-resources, might as well follow suit

Reply:
The svn bit stays annoying though, but by bundling them it'll at least not bother users anymore, just contributors.